src/fontman/font_manager.cpp

```diff
@@ -122,66 +220,46 @@ TtfFont* FontManager::getTtfFontByName(const std::string& fontName)
 #endif
 }
 
-void FontManager::initBasic()
-{
 #ifdef THEXTECH_ENABLE_TTF_SUPPORT
-    if(!g_ft)
-        initializeFreeType();
-    g_defaultTtfFont = nullptr;
+static bool s_loadFintsFromDir(const std::string &fonts_root,
```

Comment:
Advice: it's better to take an already-loaded DirListCI (`g_dirEpisode` or `g_dirCustom`, or a temporary one in the global case) as an argument to this, to avoid the expense of relisting a directory (high on some of our platforms).

Reply:
Good idea, I'll try this.

Reply:
Anyway, here is a dynamical search of files in a directory by the ".font.ini" template to find out raster fonts, and the, finding TTF fonts from a manually-filled list in the INI file.

Reply:
@ds-sloth, what do you think about this?

Reply:
Okay, I updated the thing:
- I extended the DirListCI to support necessary cals
- I reworked the font manager side to use global DirListCI instances
Should work now!